### PR TITLE
API: Moved API file directory around.

### PR DESCRIPTION
**Pull request by Liriaene**
Currently the server API code has no uniform naming scheme.

This PR is to fix that by moving and renaming files. As well as updating their references.

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "bcrypt": "^3.0.6",
     "cors": "^2.8.5",
     "express": "^4.16.4",
+    "package.json": "^2.0.1",
```
Comment (gstone271) [inline api/package.json]: What's this used for?

Comment (Liriaene) [reply api/package.json]: No idea. Removed it.

**Review by movshov — APPROVED** [review]
Aside from the merge conflict it looks good to me.

**Review by ElijahRW — APPROVED** [review]
`patient/updatesNotes.ts` should be reconfigured to `patient/notes/update.ts` to conform with the rest of the files.

**Review by OkapalDominic — COMMENTED** [review]
account and appointment looks good, just finish getting patient to conform to the convention.

**Comment by Liriaene** [comment]
Merged via #106

**Pull request closed without merging**